scripts/init.sh

```diff
@@ -2,7 +2,26 @@
 
 set -e
 
+echo "Setting up submodules"
 git submodule update --init --recursive || true
-(cd web && pnpm install)
-go install github.com/ferranbt/fastssz/sszgen@latest
+
+echo "Setting up git hooks"
 ln -sf pre-commit.sh .git/hooks/pre-commit
+
+echo "Installing Rust toolchains"
+# NOTE: This ensures that the toolchain in rust-toolchain.toml is installed.
+# rustup has no subcommand (yet) for installing the toolchain in rust-toolchain.toml:
+# https://github.com/rust-lang/rustup/issues/2686
+# The auto-installation behaviour in rustup will likely be removed:
+# https://github.com/rust-lang/rustup/issues/1397
+cp rust-toolchain-stable.toml rust-toolchain.toml
+rustup show
+cp rust-toolchain-nightly.toml rust-toolchain.toml
+rustup show
+cp rust-toolchain-stable.toml rust-toolchain.toml
```

Comment:
Hey David, what I meant on Slack is that its OK if rustup automatically installs the stable toolchain using `rust-toolchain.toml`.

These `rust-toolchain-stable.toml` and `rust-toolchain-nightly.toml` files complicate things a bit.

Reply:
So be more clear:

1. Don't install rust stable in `init.sh`, since rustup will automatically install it.

2. Install `rustfmt` in `init.sh` using `rustup toolchain install $SNOWBRIDGE_RUST_NIGHTLY`, etc.